src/models/db.js

```diff
@@ -77,6 +78,13 @@ export default function() {
       deferreds.pop().resolve(db);
     }
     state = READY;
+
+    // Load default permissions.
+    const permissions = config.get('permissions').map(permission => {
```

Comment:
Yeah I think it's better; so that we can also ship it in git.

Alternatively you can provide the default list in the `default` property for this, but I find it's clumsy.